Microsoft.Toolkit.Uwp.UI.Controls/project.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
   "dependencies": {
     "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",
+    "Robmikh.CompositionSurfaceFactory": "0.7.0-alpha",
```

Comment:
Sorry I missed this question. Can you update this file: https://github.com/Microsoft/UWPCommunityToolkit/blob/dev/build/Microsoft.Toolkit.Uwp.UI.Controls.nuspec